### PR TITLE
[css-ui] Fix box-sizing web platform test failures

### PR DESCRIPTION
**Pull request by karlcow**
#### 21c66fcd81750f21944d2057c20c00d2633011b6
<pre>
[css-ui] Fix box-sizing web platform test failures
<a href="https://bugs.webkit.org/show_bug.cgi?id=175287">https://bugs.webkit.org/show_bug.cgi?id=175287</a>
rdar://103700285

Reviewed by Tim Nguyen.

This fixes the logicalWidth computation in RenderReplaced according
to the equation described in the CSS 2.2 specification
<a href="https://www.w3.org/TR/CSS22/visudet.html#blockwidth">https://www.w3.org/TR/CSS22/visudet.html#blockwidth</a>

This also removes two tests and their expectations from the tree
which are covered by WPT tests and have also been removed
from the blink codebase. That would fix
<a href="https://bugs.webkit.org/show_bug.cgi?id=228211">https://bugs.webkit.org/show_bug.cgi?id=228211</a>

* LayoutTests/css2.1/20110323/replaced-intrinsic-003.htm: Removed.
* LayoutTests/css2.1/20110323/resources/replaced-intrinsic-003.svg: Removed.
* LayoutTests/imported/w3c/web-platform-tests/css/css-position/position-absolute-replaced-minmax-expected.txt:
* LayoutTests/platform/glib/css2.1/20110323/replaced-intrinsic-003-expected.txt: Removed.
* LayoutTests/platform/gtk/css2.1/20110323/replaced-intrinsic-003-expected.png: Removed.
* LayoutTests/platform/gtk/svg/hixie/intrinsic/003-expected.png: Removed.
* LayoutTests/platform/gtk/svg/hixie/intrinsic/003-expected.txt: Removed.
* LayoutTests/platform/ios-wk2/svg/hixie/intrinsic/003-expected.png: Removed.
* LayoutTests/platform/ios-wk2/TestExpectations:
* LayoutTests/platform/ios/css2.1/20110323/replaced-intrinsic-003-expected.txt: Removed.
* LayoutTests/platform/ios/svg/hixie/intrinsic/003-expected.txt: Removed.
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-ventura-wk2-lbse-text/svg/hixie/intrinsic/003-expected.txt: Removed.
* LayoutTests/platform/mac-ventura-wk2-pixel/svg/hixie/intrinsic/003-expected.png: Removed.
* LayoutTests/platform/mac-wk1/TestExpectations:
* LayoutTests/platform/mac/css2.1/20110323/replaced-intrinsic-003-expected.png: Removed.
* LayoutTests/platform/mac/css2.1/20110323/replaced-intrinsic-003-expected.txt: Removed.
* LayoutTests/platform/mac/svg/hixie/intrinsic/003-expected.png: Removed.
* LayoutTests/platform/mac/svg/hixie/intrinsic/003-expected.txt: Removed.
* LayoutTests/platform/win/css2.1/20110323/replaced-intrinsic-003-expected.txt: Removed.
* LayoutTests/platform/win/svg/hixie/intrinsic/003-expected.png: Removed.
* LayoutTests/platform/win/svg/hixie/intrinsic/003-expected.txt: Removed.
* LayoutTests/platform/win/TestExpectations:
* LayoutTests/platform/wincairo/css2.1/20110323/replaced-intrinsic-003-expected.txt: Removed.
* LayoutTests/platform/wincairo/svg/hixie/intrinsic/003-expected.txt: Removed.
* LayoutTests/platform/wpe/svg/hixie/intrinsic/003-expected.txt: Removed.
* LayoutTests/svg/hixie/intrinsic/003.html: Removed.
* LayoutTests/svg/hixie/intrinsic/resources/003.svg: Removed.
* LayoutTests/TestExpectations:
* Source/WebCore/rendering/RenderReplaced.cpp:
(WebCore::RenderReplaced::computeConstrainedLogicalWidth const):

Canonical link: <a href="https://commits.webkit.org/258342@main">https://commits.webkit.org/258342@main</a>
</pre>

https://github.com/WebKit/WebKit/commit/dac158050e19c0108801855df1fc3a83ceb964d9

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/101561 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/10716 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/34620 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/110864 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/171093 "Built successfully and passed tests") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/105541 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/11671 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/1618 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/93954 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/108652 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/107343 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/8892 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/92138 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/35417 "Passed tests") | 
| | [❌ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/90786 "Found 1 new API test failure: TestWebKitAPI.WebKit.QuotaDelegateReload (failure)") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/23554 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/78415 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/4314 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/25056 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/4383 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/1512 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/10469 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/44544 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/5719 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/6143 "Built successfully") | | | 
| | | | |

### CODE REVIEW

**Comment by karlcow** [comment]
Draft PR to see if the modification really fixes the same WPT tests that are fixed in my local run.

**Comment by karlcow** [comment]
This seems to fix 

http://wpt.live/css/css-sizing/aspect-ratio/replaced-element-039.html
http://wpt.live/css/css-sizing/aspect-ratio/replaced-element-040.html
http://wpt.live/css/css-sizing/replaced-aspect-ratio-stretch-fit-001.html
http://wpt.live/css/css-sizing/replaced-aspect-ratio-stretch-fit-002.html

Maybe:
http://wpt.live/html/rendering/non-replaced-elements/the-hr-element-0/setting-overflow-visible.html

Partially fixes
http://wpt.live/css/css-sizing/intrinsic-percent-non-replaced-004.html

**Comment by karlcow** [comment]
Found 3 new test failures (gtk-wk2): 

* css2.1/20110323/replaced-intrinsic-003.htm, 
* svg/hixie/intrinsic/003.html

These two are the same test. Should one or the two be removed?

The test "fails" now, but it gives the same result than other browsers.
Screenshot 2022-12-26 at 12 07 32


* imported/w3c/web-platform-tests/css/css-position/position-absolute-replaced-minmax.html, 

It fixes in my build: IMG 35
http://wpt.live/css/css-position/position-absolute-replaced-minmax.html
https://ews-build.s3-us-west-2.amazonaws.com/GTK-WK2-Tests-EWS/1496f568-35402/imported/w3c/web-platform-tests/css/css-position/position-absolute-replaced-minmax-pretty-diff.html

**Comment by karlcow** [comment]
in my local tests


```
Expected to fail, but passed: (74)
  imported/w3c/web-platform-tests/cookies/attributes/path.html
  imported/w3c/web-platform-tests/cookies/value/value.html
  imported/w3c/web-platform-tests/css/css-backgrounds/background-attachment-margin-root-001.html
  imported/w3c/web-platform-tests/css/css-backgrounds/background-clip-content-box-002.html
  imported/w3c/web-platform-tests/css/css-backgrounds/background-margin-root.html
  imported/w3c/web-platform-tests/css/css-backgrounds/background-size/background-size-contain.xht
  imported/w3c/web-platform-tests/css/css-backgrounds/background-size/background-size-cover-svg.html
  imported/w3c/web-platform-tests/css/css-backgrounds/background-size/background-size-cover.xht
  imported/w3c/web-platform-tests/css/css-backgrounds/border-image-repeat-round-1.html
  imported/w3c/web-platform-tests/css/css-backgrounds/border-radius-clip-001.html
  imported/w3c/web-platform-tests/css/css-backgrounds/css3-border-image-repeat-repeat.html
  imported/w3c/web-platform-tests/css/css-color/deprecated-sameas-017.html
  imported/w3c/web-platform-tests/css/css-color/deprecated-sameas-018.html
  imported/w3c/web-platform-tests/css/css-color/deprecated-sameas-019.html
  imported/w3c/web-platform-tests/css/css-color/system-color-compute.html
  imported/w3c/web-platform-tests/css/css-color/system-color-consistency.html
  imported/w3c/web-platform-tests/css/css-color/t422-rgba-onscreen-b.xht
  imported/w3c/web-platform-tests/css/css-color/t422-rgba-onscreen-multiple-boxes-c.xht
  imported/w3c/web-platform-tests/css/css-color/t425-hsla-onscreen-b.xht
  imported/w3c/web-platform-tests/css/css-color/t425-hsla-onscreen-multiple-boxes-c.xht
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-background-properties.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-default.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-composited-dynamic1.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-composited-dynamic2.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-composited.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-dynamic1.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-dynamic2.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-none-image-document.html
  imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-none.html
  imported/w3c/web-platform-tests/css/css-images/multiple-position-color-stop-conic-2.html
  imported/w3c/web-platform-tests/css/css-images/tiled-conic-gradients.html
  imported/w3c/web-platform-tests/css/css-masking/clip-path/clip-path-polygon-013.html
  imported/w3c/web-platform-tests/css/css-multicol/zero-column-width-layout.html
  imported/w3c/web-platform-tests/css/css-position/multicol/static-position/vlr-ltr-ltr-in-multicol.html
  imported/w3c/web-platform-tests/css/css-position/multicol/static-position/vrl-ltr-ltr-in-multicol.html
  imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-004.html
  imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-computed.html
  imported/w3c/web-platform-tests/css/css-pseudo/placeholder-excluded-properties.html
  imported/w3c/web-platform-tests/css/css-transforms/individual-transform/animation/individual-transform-ordering.html
  imported/w3c/web-platform-tests/css/css-typed-om/rotate-by-added-angle.html
  imported/w3c/web-platform-tests/css/css-typed-om/width-by-clamp-px-em.html
  imported/w3c/web-platform-tests/css/css-typed-om/width-by-max-px-em.html
  imported/w3c/web-platform-tests/css/css-typed-om/width-by-min-px-em.html
  imported/w3c/web-platform-tests/css/css-ui/appearance-menulist-button-002.tentative.html
  imported/w3c/web-platform-tests/css/css-ui/webkit-appearance-menulist-button-002.tentative.html
  imported/w3c/web-platform-tests/css/css-writing-modes/float-lft-orthog-htb-in-vlr-002.xht
  imported/w3c/web-platform-tests/css/css-writing-modes/float-rgt-orthog-htb-in-vlr-003.xht
  imported/w3c/web-platform-tests/css/cssom/css-style-attr-decl-block.html
  imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-opacity.html
  imported/w3c/web-platform-tests/css/filter-effects/effect-reference-displacement-negative-scale-001.html
  imported/w3c/web-platform-tests/css/filter-effects/effect-reference-local-url-with-base-001.html
  imported/w3c/web-platform-tests/css/filter-effects/effect-reference-rename-002.html
  imported/w3c/web-platform-tests/css/filter-effects/fecomposite-non-zero-inoffset-001.html
  imported/w3c/web-platform-tests/css/filter-effects/svg-mutation-function-to-url.html
  imported/w3c/web-platform-tests/html/canvas/element/line-styles/2d.line.miter.obtuse.html
  imported/w3c/web-platform-tests/html/canvas/element/manual/drawing-images-to-the-canvas/image-orientation/drawImage-from-blob.tentative.html
  imported/w3c/web-platform-tests/html/canvas/element/manual/drawing-text-to-the-canvas/canvas.2d.disconnected.html
  imported/w3c/web-platform-tests/html/canvas/element/manual/imagebitmap/imageBitmapRendering-transferFromImageBitmap-webgl.html
  imported/w3c/web-platform-tests/html/canvas/element/manual/wide-gamut-canvas/canvas-display-p3-drawImage-ImageBitmap-video.html
  imported/w3c/web-platform-tests/html/canvas/element/manual/wide-gamut-canvas/canvas-display-p3-drawImage-video.html
  imported/w3c/web-platform-tests/html/canvas/element/path-objects/2d.path.arc.selfintersect.1.html
  imported/w3c/web-platform-tests/html/canvas/element/path-objects/2d.path.arc.shape.3.html
  imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-textarea-N-between-Rs.html
  imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-textarea-script-N-between-Rs.html
  imported/w3c/web-platform-tests/html/rendering/non-replaced-elements/tables/table-cell-nowrap-with-fixed-width.html
  imported/w3c/web-platform-tests/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-list-item-numbering.html
  imported/w3c/web-platform-tests/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-list-item.html
  imported/w3c/web-platform-tests/quirks/line-height-in-list-item.tentative.html
  imported/w3c/web-platform-tests/svg/import/animate-elem-35-t-manual.svg
  imported/w3c/web-platform-tests/svg/path/distance/pathlength-path.svg
  imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-relay-canvas.https.html
  imported/w3c/web-platform-tests/xhr/event-timeout-order.any.worker.html
  imported/w3c/web-platform-tests/xhr/setrequestheader-case-insensitive.htm
```

some of them show no visible differences
but some of them are solved by this patch.

File: LayoutTests/TestExpectations [hunk 1]
```diff
@@ -3128,13 +3128,13 @@ webkit.org/b/172609 fetch/closing-while-fetching-blob.html [ Skip ]
...
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-015.html [ Pass ]
 webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-016.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-018.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-019.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-024.html [ ImageOnlyFailure ]
-webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-025.html [ ImageOnlyFailure ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-018.html [ Pass ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-019.html [ Pass ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-024.html [ Pass ]
+webkit.org/b/175287 imported/w3c/web-platform-tests/css/css-ui/box-sizing-025.html [ Pass ]
```
Comment (nt1m) [inline LayoutTests/TestExpectations]: Removing these lines would be better than marking them as `Pass`

File: LayoutTests/TestExpectations [hunk 2]
```diff
@@ -5938,10 +5938,10 @@ imported/w3c/web-platform-tests/url/idlharness-shadowrealm.window.html [ Pass Fa
...
 
 # Failing since their import.
-imported/w3c/web-platform-tests/css/css-typed-om/rotate-by-added-angle.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-typed-om/width-by-clamp-px-em.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-typed-om/width-by-max-px-em.html [ ImageOnlyFailure ]
-imported/w3c/web-platform-tests/css/css-typed-om/width-by-min-px-em.html [ ImageOnlyFailure ]
+imported/w3c/web-platform-tests/css/css-typed-om/rotate-by-added-angle.html [ Pass ]
+imported/w3c/web-platform-tests/css/css-typed-om/width-by-clamp-px-em.html [ Pass ]
+imported/w3c/web-platform-tests/css/css-typed-om/width-by-max-px-em.html [ Pass ]
+imported/w3c/web-platform-tests/css/css-typed-om/width-by-min-px-em.html [ Pass ]
```
Comment (nt1m) [inline LayoutTests/TestExpectations]: ditto

File: LayoutTests/TestExpectations [hunk 3]
```diff
@@ -6200,3 +6200,7 @@ imported/w3c/web-platform-tests/css/css-nesting/conditional-properties.html [ Im
 imported/w3c/web-platform-tests/css/css-nesting/conditional-rules.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-nesting/implicit-nesting.html [ ImageOnlyFailure ]
 imported/w3c/web-platform-tests/css/css-nesting/nesting-basic.html [ ImageOnlyFailure ]
+
+# To be removed?
+webkit.org/b/228211 css2.1/20110323/replaced-intrinsic-003.htm [ Failure ]
+webkit.org/b/228211 svg/hixie/intrinsic/003.html [ Failure ]
```
Comment (nt1m) [inline LayoutTests/TestExpectations]: Probably makes sense to remove these tests from the tree. As your passes show, there are other tests covering this. They have been removed from the blink codebase as well (cs.chromium.org)

File: LayoutTests/TestExpectations (hunk 1 above)
Comment (karlcow) [reply LayoutTests/TestExpectations]: done

File: LayoutTests/TestExpectations (hunk 2 above)
Comment (karlcow) [reply LayoutTests/TestExpectations]: done

File: LayoutTests/TestExpectations (hunk 3 above)
Comment (karlcow) [reply LayoutTests/TestExpectations]: done

**Review by nt1m — APPROVED** [review]

**Comment by nt1m** [comment]
> in my local tests
> 
> ```
> Expected to fail, but passed: (74)
>   imported/w3c/web-platform-tests/cookies/attributes/path.html
>   imported/w3c/web-platform-tests/cookies/value/value.html
>   imported/w3c/web-platform-tests/css/css-backgrounds/background-attachment-margin-root-001.html
>   imported/w3c/web-platform-tests/css/css-backgrounds/background-clip-content-box-002.html
>   imported/w3c/web-platform-tests/css/css-backgrounds/background-margin-root.html
>   imported/w3c/web-platform-tests/css/css-backgrounds/background-size/background-size-contain.xht
>   imported/w3c/web-platform-tests/css/css-backgrounds/background-size/background-size-cover-svg.html
>   imported/w3c/web-platform-tests/css/css-backgrounds/background-size/background-size-cover.xht
>   imported/w3c/web-platform-tests/css/css-backgrounds/border-image-repeat-round-1.html
>   imported/w3c/web-platform-tests/css/css-backgrounds/border-radius-clip-001.html
>   imported/w3c/web-platform-tests/css/css-backgrounds/css3-border-image-repeat-repeat.html
>   imported/w3c/web-platform-tests/css/css-color/deprecated-sameas-017.html
>   imported/w3c/web-platform-tests/css/css-color/deprecated-sameas-018.html
>   imported/w3c/web-platform-tests/css/css-color/deprecated-sameas-019.html
>   imported/w3c/web-platform-tests/css/css-color/system-color-compute.html
>   imported/w3c/web-platform-tests/css/css-color/system-color-consistency.html
>   imported/w3c/web-platform-tests/css/css-color/t422-rgba-onscreen-b.xht
>   imported/w3c/web-platform-tests/css/css-color/t422-rgba-onscreen-multiple-boxes-c.xht
>   imported/w3c/web-platform-tests/css/css-color/t425-hsla-onscreen-b.xht
>   imported/w3c/web-platform-tests/css/css-color/t425-hsla-onscreen-multiple-boxes-c.xht
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-background-properties.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-default.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-composited-dynamic1.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-composited-dynamic2.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-composited.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-dynamic1.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image-dynamic2.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-from-image.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-none-image-document.html
>   imported/w3c/web-platform-tests/css/css-images/image-orientation/image-orientation-none.html
>   imported/w3c/web-platform-tests/css/css-images/multiple-position-color-stop-conic-2.html
>   imported/w3c/web-platform-tests/css/css-images/tiled-conic-gradients.html
>   imported/w3c/web-platform-tests/css/css-masking/clip-path/clip-path-polygon-013.html
>   imported/w3c/web-platform-tests/css/css-multicol/zero-column-width-layout.html
>   imported/w3c/web-platform-tests/css/css-position/multicol/static-position/vlr-ltr-ltr-in-multicol.html
>   imported/w3c/web-platform-tests/css/css-position/multicol/static-position/vrl-ltr-ltr-in-multicol.html
>   imported/w3c/web-platform-tests/css/css-pseudo/highlight-cascade-004.html
>   imported/w3c/web-platform-tests/css/css-pseudo/highlight-pseudos-computed.html
>   imported/w3c/web-platform-tests/css/css-pseudo/placeholder-excluded-properties.html
>   imported/w3c/web-platform-tests/css/css-transforms/individual-transform/animation/individual-transform-ordering.html
>   imported/w3c/web-platform-tests/css/css-typed-om/rotate-by-added-angle.html
>   imported/w3c/web-platform-tests/css/css-typed-om/width-by-clamp-px-em.html
>   imported/w3c/web-platform-tests/css/css-typed-om/width-by-max-px-em.html
>   imported/w3c/web-platform-tests/css/css-typed-om/width-by-min-px-em.html
>   imported/w3c/web-platform-tests/css/css-ui/appearance-menulist-button-002.tentative.html
>   imported/w3c/web-platform-tests/css/css-ui/webkit-appearance-menulist-button-002.tentative.html
>   imported/w3c/web-platform-tests/css/css-writing-modes/float-lft-orthog-htb-in-vlr-002.xht
>   imported/w3c/web-platform-tests/css/css-writing-modes/float-rgt-orthog-htb-in-vlr-003.xht
>   imported/w3c/web-platform-tests/css/cssom/css-style-attr-decl-block.html
>   imported/w3c/web-platform-tests/css/filter-effects/css-filters-animation-opacity.html
>   imported/w3c/web-platform-tests/css/filter-effects/effect-reference-displacement-negative-scale-001.html
>   imported/w3c/web-platform-tests/css/filter-effects/effect-reference-local-url-with-base-001.html
>   imported/w3c/web-platform-tests/css/filter-effects/effect-reference-rename-002.html
>   imported/w3c/web-platform-tests/css/filter-effects/fecomposite-non-zero-inoffset-001.html
>   imported/w3c/web-platform-tests/css/filter-effects/svg-mutation-function-to-url.html
>   imported/w3c/web-platform-tests/html/canvas/element/line-styles/2d.line.miter.obtuse.html
>   imported/w3c/web-platform-tests/html/canvas/element/manual/drawing-images-to-the-canvas/image-orientation/drawImage-from-blob.tentative.html
>   imported/w3c/web-platform-tests/html/canvas/element/manual/drawing-text-to-the-canvas/canvas.2d.disconnected.html
>   imported/w3c/web-platform-tests/html/canvas/element/manual/imagebitmap/imageBitmapRendering-transferFromImageBitmap-webgl.html
>   imported/w3c/web-platform-tests/html/canvas/element/manual/wide-gamut-canvas/canvas-display-p3-drawImage-ImageBitmap-video.html
>   imported/w3c/web-platform-tests/html/canvas/element/manual/wide-gamut-canvas/canvas-display-p3-drawImage-video.html
>   imported/w3c/web-platform-tests/html/canvas/element/path-objects/2d.path.arc.selfintersect.1.html
>   imported/w3c/web-platform-tests/html/canvas/element/path-objects/2d.path.arc.shape.3.html
>   imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-textarea-N-between-Rs.html
>   imported/w3c/web-platform-tests/html/dom/elements/global-attributes/dir_auto-textarea-script-N-between-Rs.html
>   imported/w3c/web-platform-tests/html/rendering/non-replaced-elements/tables/table-cell-nowrap-with-fixed-width.html
>   imported/w3c/web-platform-tests/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-list-item-numbering.html
>   imported/w3c/web-platform-tests/html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/legend-list-item.html
>   imported/w3c/web-platform-tests/quirks/line-height-in-list-item.tentative.html
>   imported/w3c/web-platform-tests/svg/import/animate-elem-35-t-manual.svg
>   imported/w3c/web-platform-tests/svg/path/distance/pathlength-path.svg
>   imported/w3c/web-platform-tests/webrtc/RTCPeerConnection-relay-canvas.https.html
>   imported/w3c/web-platform-tests/xhr/event-timeout-order.any.worker.html
>   imported/w3c/web-platform-tests/xhr/setrequestheader-case-insensitive.htm
> ```
> 
> some of them show no visible differences but some of them are solved by this patch.

Would be nice to find out which ones are fixed by this patch. You can make a separate follow up PR that removes all of these for the TestExpectations, run them through EWS, and then put them back as you see failures.
Manual inspection works too, up to you.

**Comment by karlcow** [comment]
> Would be nice to find out which ones are fixed by this patch. You can make a separate follow up PR that removes all of these for the TestExpectations, run them through EWS, and then put them back as you see failures. Manual inspection works too, up to you.

yes that would be great. I will add it to my TODO.
Let's open a bug for it.

**Comment by karlcow** [comment]
> > Would be nice to find out which ones are fixed by this patch. You can make a separate follow up PR that removes all of these for the TestExpectations, run them through EWS, and then put them back as you see failures. Manual inspection works too, up to you.
> 
> yes that would be great. I will add it to my TODO. Let's open a bug for it.

Opened https://bugs.webkit.org/show_bug.cgi?id=249901

**Pull request merged**